parallel_programming/Parallel.py

```diff
@@ -0,0 +1,62 @@
+import time
+from multiprocessing import Process, Manager  # , Queue
+import itertools
+import argparse
+
+
+def fasta_reader(fasta_name):
+    with open(fasta_name, "r") as file:
+        seq = []
+        for line in file:
+            if line.startswith(">"):
+                contig = line.split()[0][1:]
+            else:
+                seq = line.replace("\n", "")
+                unique = dict(zip(list(seq), [list(seq).count(i) for i in list(seq)]))
```

Comment:
Как альтернативный вариант
```suggestion
                unique = Counter(seq)
```
`Counter` это функция из модуля `collections`, которая делает всю "магию" по подсчёту букв